### DISCUSSION TITLE
Click on a character?

### DISCUSSION THREAD

**Discussion by timelord666** [dialogic-godot/dialogic#1564] [Q&A]
How to register a click on a dialogic character? I want just add characters via dialogic and if i click on desired character make something like animation

**Comment by Jowan-Spooner** [comment]
Hi @timelord666. This sounds like a good moment for a custom portrait scene. What version of dialogic are you using?

**Reply by timelord666** [reply]
Hi, 1.4.5

**Reply by timelord666** [reply]
i tried to add scene portrait but it isn't showing up(
